integration-testing/http_client_test.py

```diff
@@ -82,7 +82,7 @@ def test_simple_get_google(self):
         run_command(simple_get_args)
     def test_simple_get_h1(self):
         """make a simple GET request via HTTP/1.1 and make sure it succeeds"""
-        simple_get_args = elasticurl_cmd_prefix + ['-v', 'TRACE', '--http1_1', 'http://example.com']
```

Comment:
using `httpbin.org` instead of `www.amazon.com` here because there's a separate test `test_simple_get_amazon` that explicitly hits `www.amazon.com`. Other tests were already hitting `httpbin.org` so this doesn't add any NEW dependencies